mitmproxy/utils/typecheck.py

```diff
@@ -1,7 +1,7 @@
 import typing
 
 
-def check_type(name: str, value: typing.Any, typeinfo: type) -> None:
+def check_type(name: str, value: typing.Any, typeinfo: typing.Any) -> None:
```

Comment:
This makes underscore attribute access on the type object in the method barf. We can tackle this a different way by peppering the function body with ignore annotations.